synapse/models/infotech.py

```diff
@@ -934,6 +956,92 @@ def getModelDefs(self):
+
+                    ('alert:meantime:triage', ('duration', {}), {
+                        'doc': 'The mean time to triage for alerts generated within the time window.'}),
+
+                    ('alert:count', ('int', {}), {
+                        'doc': 'The total number of alerts generated within the time window.'}),
+
+                    ('alert:falsepos', ('int', {}), {
+                        'doc': 'The number of alerts generated within the time window that were determined to be false positives.'}),
+
+                    ('assets:hosts', ('int', {}), {
+                        'doc': 'The total number of hosts within scope for the information security program.'}),
+
+                    ('assets:users', ('int', {}), {
+                        'doc': 'The total number of users within scope for the information security program.'}),
+
+                    ('assets:vulns:existing', ('int', {}), {
+                        'doc': 'The number of asset vulnerabilities being tracked at the beginning of the window.'}),
+
+                    ('assets:vulns:discovered', ('int', {}), {
+                        'doc': 'The number of asset vulnerabilities discovered during the window.'}),
+
+                    ('assets:vulns:mitigated', ('int', {}), {
+                        'doc': 'The number of asset vulnerabilities mitigated during the window.'}),
+
+                    ('assets:vulns:meantime:mitigate', ('duration', {}), {
+                        'doc': 'The mean time to mitigate for vulnerable assets mitigated during the time window.'}),
+
+                )),
+
+                ('it:sec:vuln:scan', {}, (
+
+                    ('time', ('time', {}), {
+                        'doc': 'The time that the scan was started.'}),
+
+                    ('desc', ('str', {}), {
+                        'doc': 'Description of the scan and scope.'}),
+
+                    ('soft', ('it:prod:softver', {}), {
+                        'doc': 'The scanning software used.'}),
+
+                    ('soft:name', ('it:prod:softname', {}), {
+                        'doc': 'The name of the scanner software.'}),
+
+                    ('operator', ('ps:contact', {}), {
+                        'doc': 'Contact information for the scan operator.'}),
+
+                )),
+
+                ('it:sec:vuln:scan:hit', {}, (
+
+                    ('scan', ('risk:vuln:scan', {}), {
+                        'doc': 'The scan that discovered the vulnerability in the asset.'}),
+
+                    ('vuln', ('risk:vuln', {}), {
+                        'doc': 'The vulnerability detected in the asset.'}),
+
+                    ('asset', ('ndef', {}), {
+                        'doc': 'The node which is vulnerable.'}),
+
+                    ('desc', ('str', {}), {
+                        'doc': 'A description of the vulnerabilty and how it was detected in the asset.'}),
+
+                    ('mitigated', ('time', {}), {
```

Comment:
How to link the (specific) mitigation to the mitigated asset? We currently have risk:mitigation -(addresses)> ou:technique but that feels more abstract. orgs may want to know the specific/tactical action someone took to "fix" a particular asset - whether "apply vendor patch" or "change default password" or "temporarily disable vulnerable service". Feels like a gap (gaps?) but maybe we just need a :mitigation prop on this node?

Reply:
Woke up thinking the same thing. I'll add a `:mitigation` field :+1: